src/widgets/helper/NotebookTab.hpp

```diff
@@ -107,6 +122,25 @@ class NotebookTab : public Button
 
     int normalTabWidthForHeight(int height) const;
 
+    bool shouldMessageHighlight(const ChannelView &channelViewSource,
+                                const MessagePtr &message) const;
+
+    struct HighlightSources {
+        std::unordered_set<ChannelView::ChannelViewID> newMessageSource;
+        std::unordered_set<ChannelView::ChannelViewID> highlightedSource;
```

Comment:
Document these members - I'm not sold on the naming of `highlight source`, so my thought is that with these documented to better describe your intent I might understand whether the name is ok or not

Reply:
the source in here refers to the split in which the message appears, I'll add this as a comment there